src/DataAccess/Entities/Apprenticeship/EpisodePrice.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace SFA.DAS.Apprenticeships.DataAccess.Entities.Apprenticeship;
+
+[Table("dbo.EpisodePrice")]
+[System.ComponentModel.DataAnnotations.Schema.Table("EpisodePrice")]
+public class EpisodePrice
+{
```

Comment:
More inconsistent indentation

Reply:
It's a mixture of tabs and spaces I think which causes it to be funky or non funky depending on which viewer you are using and if it decides a tab is the same width as however many spaces it's used.